src/libraries/System.Runtime.Serialization.Schema/src/System.Runtime.Serialization.Schema.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IsTrimmable>false</IsTrimmable>
+    <IsPackable>true</IsPackable>
```

Comment:
Same comment about timmability here. Is this package really not trimming safe?

Reply:
Oops. Had that in there just to keep things simple when starting the project from scratch. We should remove it.